README.md

```diff
@@ -19,9 +19,9 @@ Requirements specific to Azure AI Content Safety embedded SDK samples are as fol
 
 ## Build the sample
 
-**By building this sample you will download the Microsoft Azure AI Content Safety embedded SDK. By downloading you acknowledge its license, see [Azure AI Content Safety embedded SDK license agreement](//license agreement link).**
+**By building this sample you will download the Microsoft Azure AI Content Safety embedded SDK. By downloading you acknowledge its license, see [Azure AI Content Safety embedded SDK license agreement](https://github.com/Azure/azure-ai-content-safety-sdk/blob/main/LICENSE.txt).**
 
```

Comment:
This LICENSE.txt applies only to the sample code repository, not to the Embedded SDK, which has its license included in the NuGet package.